modules/statics/src/coins.ts

```diff
@@ -329,6 +331,7 @@ export const coins = CoinMap.fromCoins([
   erc20('cel', 'Celsius', 4, '0xaaaebe6fe48e54f431b0c390cfaf0b017d09d42d', UnderlyingAsset.CEL),
   erc20('celr', 'Celer Network', 18, '0x4f9254c83eb525f9fcf346490bbb3ed28a81c667', UnderlyingAsset.CELR),
   erc20('ceth', 'Compound Ether', 8, '0x4ddc2d193948926d02f9b1fe9e1daa0718270ed5', UnderlyingAsset.CETH),
+  erc20('cho', 'Choise.com', 8, '0xbba39fd2935d5769116ce38d46a71bde9cf03099', UnderlyingAsset.CHO),
```

Comment:
Same with this one. It's listed as lowercase `choise.com Token`.

Reply:
ok